frontend/src/metabase/search/components/SearchFilterDatePicker/SearchFilterDatePicker.styled.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from "@emotion/styled";
+import { DateAllOptionsWidget } from "metabase/components/DateAllOptionsWidget";
+
+export const SearchFilterDatePickerWrapper = styled(DateAllOptionsWidget)`
```

Comment:
Please don't change or use dashboard components here. Could you make it work using `DatePicker` only?